libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -1520,3 +1523,62 @@ uint8_t AP_AHRS_NavEKF::get_primary_gyro_index(void) const
 
 #endif // AP_AHRS_NAVEKF_AVAILABLE
 
+/*
+ * Update AOA and SSA estimation based on airspeed, velocity vector and wind vector
+ *
+ * Based on:
+ * "On estimation of wind velocity, angle-of-attack and sideslip angle of small UAVs using standard sensors" by
+ * Tor A. Johansen, Andrea Cristofaro, Kim Sorensen, Jakob M. Hansen, Thor I. Fossen
+ *
+ * "Multi-Stage Fusion Algorithm for Estimation of Aerodynamic Angles in Mini Aerial Vehicle" by
+ * C.Ramprasadh and Hemendra Arya
+ *
+ * "ANGLE OF ATTACK AND SIDESLIP ESTIMATION USING AN INERTIAL REFERENCE PLATFORM" by
+ * JOSEPH E. ZEIS, JR., CAPTAIN, USAF
+ */
+void AP_AHRS_NavEKF::update_AOA_SSA(void)
```

Comment:
is there some reason to have this in AP_AHRS_NavEKF? I think it could be generic in AP_AHRS, and be used for all backends. That will allow it to work in XPlane10 and RealFlight for example (which use EKF 10, and not AP_AHRS_NavEKF)